doc/mapping/nts.md

```diff
@@ -41,6 +41,17 @@ public class City
 var nearbyCities = context.Cities.Where(c => c.Location.Distance(somePoint) < 100);
 ```
 
+## Constraining your type names
+
+With the code above, the provider will create a database column of type `geometry`. This is perfectly fine, but be aware that this type accepts any geometry type (point, polygon...), with any coordinate system (XY, XYZ...). It's good practice to constrain the colun to exact type of data you will be storing, but unfortunately the provider isn't aware of your required coordinate system and therefore can't do that for you. Consider explicitly specifying your column types on your properties as follows:
```

Comment:
Typo: "constrain the colun to exact type" should be "constrain the _column_ to _the_ exact type"